src/GraphQL/Utilities/Visitors/AppliedDirectivesValidationVisitor.cs

```diff
@@ -7,23 +7,27 @@ namespace GraphQL.Utilities
     /// <summary>
     /// This visitor verifies the correct application of directives to the schema elements.
     /// </summary>
-    internal sealed class AppliedDirectivesValidationVisitor : ISchemaNodeVisitor
+    public sealed class AppliedDirectivesValidationVisitor : ISchemaNodeVisitor
```

Comment:
I have already said that I see it wrong. This is an artificial extension of the public API. For now, keep it public, OK. This is the lesser evil now.